website/docs/d/nsxt_edgegateway.html.markdown

```diff
@@ -10,29 +10,52 @@ description: |-
 
 Provides a VMware Cloud Director NSX-T edge gateway data source. This can be used to read NSX-T edge gateway configurations.
 
--> **Note:** This data source uses new VMware Cloud Director
-[OpenAPI](https://code.vmware.com/docs/11982/getting-started-with-vmware-cloud-director-openapi) and
-requires at least VCD *10.1.1+* and NSX-T *3.0+*.
-
 Supported in provider *v3.1+*.
 
-## Example Usage 
+## Example Usage (NSX-T Edge Gateway belonging to VDC group)
 
 ```hcl
+data "vcd_vdc_group" "group1" {
+  name = "existing-group"
+}
+
 data "vcd_nsxt_edgegateway" "t1" {
-  org  = "myorg"
-  vdc  = "my-nsxt-vdc"
-  name = "nsxt-edge-gateway"
+  org      = "myorg"
+  owner_id = data.vcd_vdc_group.group1.id
+  name     = "nsxt-edge-gateway"
 }
 ```
 
+## Example Usage (NSX-T Edge Gateway belonging to VDC)
+
+```hcl
+data "vcd_org_vdc" "vdc1" {
+  name = "existing-vdc"
+}
+
+data "vcd_nsxt_edgegateway" "t1" {
+  org      = "myorg"
+  owner_id = data.vcd_org_vdc.vdc1.id
+  name     = "nsxt-edge-gateway"
+}
+```
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `org` - (Optional) The name of organization to which the edge gatweway belongs. Optional if defined at provider level.
-* `vdc` - (Optional) The name of VDC that owns the edge gateway. Optional if defined at provider level.
+* `org` - (Optional) The name of organization to which the NSX-T Edge Gateway belongs. Optional if
+  defined at provider level.
+* `vdc` - (Optional)  **Deprecated** - please use `owner_id` field. The name of VDC that owns the
+  NSX-T Edge Gateway. Optional if defined at provider level.
+* `owner_id` - (Optional, *v3.6+*,*VCD 10.2+*) **Replaces** `vdc` field. The ID of VDC or VDC group
+that this Edge Gateway belongs to. **Note:** Data source
+[vcd_vdc_group](/providers/vmware/vcd/latest/docs/data-sources/vdc_group) can be used to lookup ID
+by name.
```

Comment:
At least GitHub shows the indentation of the above three lines lost.

Reply:
It has to be so. It is just a "line wrap", but still goes under the same bullet point in markdown preview:

<img width="716" alt="image" src="https://user-images.githubusercontent.com/15804230/158319452-1a26bf10-cfe9-491d-8330-0832a57d5253.png">